react/src/views/Transfers/CreateTransferAgreement/CreateTransferAgreementView.tsx

```diff
@@ -77,7 +78,26 @@ function CreateTransferAgreementView() {
   const [createTransferAgreementMutation, createTransferAgreementMutationState] = useMutation<
     CreateTransferAgreementMutation,
     CreateTransferAgreementMutationVariables
-  >(CREATE_AGREEMENT_MUTATION);
+  >(CREATE_AGREEMENT_MUTATION, {
+    update(cache, { data: returnedTransferAgreement }) {
```

Comment:
@vahidbazzaz I tried out here how to directly write to the cache instead of using the refetchqueries.
Here, the [reference](https://www.apollographql.com/docs/react/data/mutations/#the-update-function).